examples/battery_pool.py

```diff
@@ -81,7 +28,7 @@ async def main() -> None:
         host=HOST, port=PORT, resampler_config=ResamplerConfig(resampling_period_s=1.0)
     )
 
-    battery_pool = create_battery_pool()
+    battery_pool = microgrid.battery_pool()
     receivers: Dict[str, Receiver[Any]] = {
```

Comment:
Have you run the example with these changes? just checking...

Reply:
yes,  it is able to use the battery_pool instance from the microgrid module.